test/console_test.exs

```diff
@@ -0,0 +1,98 @@
+defmodule ConsoleTest do
+  use ExUnit.Case
+  import ExUnit.CaptureIO
+
+  doctest Console
+
+  describe "when there are no markers on the board" do
+    test "#print_board" do
+      board = [0, 1, 2, 3, 4, 5, 6, 7, 8]
+      test_output = fn -> 
+        Console.print_board(board)
+      end
+      assert capture_io(test_output) == """
+     
+      --------------
+  
+        0   1   2
+       ===+===+=== 
+        3   4   5
+       ===+===+=== 
+        6   7   8
+  
+      --------------
+  
+      """
+    end
+  end
+
+  describe "when there is a marker on the board" do
+    test "#print_board" do
+      board = [0, 1, 2, 3, 4, 5, 6, 7, 8]
+      test_output = fn -> 
+        Player.move(board, 4) |> Console.print_board
+      end
+      assert capture_io(test_output) == """
+
+      --------------
+  
+        0   1   2
+       ===+===+=== 
+        3   X   5
+       ===+===+=== 
+        6   7   8
+  
+      --------------
+      
+      """
+    end
+    
+  end
+
+  describe "when the game starts running" do
+    test "#print_welcome_msg" do
+      test_output = fn -> 
+        Console.print_welcome_msg
+      end
+      assert capture_io(test_output) == """
+      Tic Tac Toe
+      -----------
+      Instructions:
+      Select the spot you want to mark by entering the corresponding number.
+      The first to get three marks in a row will win the game.
+
+      You are player "X", please start the game.
+      """
```

Comment:
If you extract your messages, it is fine to test against this file rather than hard-coding the expected output. This will make the tests less breakable if you change your messages.